src/commands/starsystems.py

```diff
@@ -0,0 +1,35 @@
+from ..packages.context import Context
+from ..packages.commands import command
+from ..packages import permissions
+from loguru import logger
+
+
+@permissions.require_permission(permissions.RAT)
+@command("search")
+async def cmd_search(ctx: Context):
+    if len(ctx.words) != 2:
+        return await ctx.reply("Usage: search <name of system>")
+    results = await ctx.bot.galaxy.search_systems_by_name(ctx.words[1])
+    return await ctx.reply(f"{results!r}")
+
+
+async def cmd_landmark_near(ctx: Context):
+    if len(ctx.words) < 2:
+        return await ctx.reply("Usage: landmark near <system>")
+
+    system_name = ctx.words_eol[1]
+    logger.trace("searching for system {}", system_name)
+    found = await ctx.bot.galaxy.find_system_by_name(system_name)
+    if not found:
+        return await ctx.reply(f"{system_name} was not found in The Fuel Rats System Database.")
+
+    logger.debug("found system {}, acquiring nearest landmark...", found)
+    nearest_landmark = await ctx.bot.galaxy.find_nearest_landmark(found)
+    return await ctx.reply(f"{nearest_landmark}")
```

Comment:
[Suggestion] `find_nearest_landmark` also returns the distance as a second return value, suggest we pass this info along.

Reply:
[error] actually implement this routine properly.
whats currently here is test code